fee_calculator/apps/calculator/fixtures/scenario.yaml

```diff
@@ -27,25 +27,13 @@
   fields: {name: Breach of crown court order}
 - model: calculator.scenario
   pk: 10
-  fields: {name: Cracked before retrial (first third)}
+  fields: {name: Cracked before retrial}
 - model: calculator.scenario
   pk: 11
   fields: {name: Retrial}
 - model: calculator.scenario
   pk: 12
```

Comment:
Further to our discussion about a coming PR to have an additional `code` attribute that uses the equivalent CCR/CCLF identifier for the scenario this is irrelevant but noting for posterity that 'Elected cases not proceeded' maps to AS000014 in CCR and ST4TS0T1 in CCLF